core/tests/dispatch_integration_tests.rs

```diff
@@ -600,85 +613,73 @@ fn remote_delivery_to_registered_actors_lazy_mixed_udp() {
 #[ignore]
 fn remote_lost_and_continued_connection() {
     let mut net_cfg = NetworkConfig::default();
-    net_cfg.set_max_connection_retry_attempts(6);
-    net_cfg.set_connection_retry_interval(500);
-    let system = system_from_network_config(net_cfg);
-    let remote_a = system_from_network_config(NetworkConfig::default());
-    let remote_port = remote_a.system_path().port();
-
-    //let (ponger_unique, pouf) = remote.create_and_register(PongerAct::new);
-    let (ponger_named, ponf) = remote_a.create_and_register(PongerAct::new_lazy);
-    let poaf = remote_a.register_by_alias(&ponger_named, "custom_name");
-    ponf.wait_expect(Duration::from_millis(1000), "Ponger failed to register!");
-    poaf.wait_expect(Duration::from_millis(1000), "Ponger failed to register!");
+    let ping_interval = Duration::from_millis(100);
+
+    net_cfg.set_max_connection_retry_attempts(CONNECTION_RETRY_ATTEMPTS);
+    net_cfg.set_connection_retry_interval(CONNECTION_RETRY_INTERVAL);
+    let pinger_system = system_from_network_config(net_cfg);
+    let ponger_system_1 = system_from_network_config(NetworkConfig::default());
+    let ponger_system_port = ponger_system_1.system_path().port();
+
+    let (ponger_named, _) = start_ponger(&ponger_system_1, PongerAct::new_lazy());
+    ponger_system_1
+        .register_by_alias(&ponger_named, "custom_name")
+        .wait_expect(REGISTRATION_TIMEOUT, "Ponger failed to register!");
     let named_path = ActorPath::Named(NamedPath::with_system(
-        remote_a.system_path(),
+        ponger_system_1.system_path(),
```

Comment:
I think Rust style dictates no _ before numerals.

Reply:
I can only find [this](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html) recommendation:

> In snake_case or SCREAMING_SNAKE_CASE, a "word" should never consist of a single letter unless it is the last "word". So, we have btree_map rather than b_tree_map, but PI_2 rather than PI2.

Reply:
Ok, looks like I was wrong then. Nevermind :)